content/account-and-profile/setting-up-and-managing-your-github-profile/index.md

```diff
@@ -1,4 +1,4 @@
----
```

Comment:
-the-codeql-workflow.md-the-codeql-workflow.mdcontent/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/troubleshooting-the-codeql-workflow.mdhttps://github.com/testdouble/standard$ bundle exec standardrb
standard: Use Ruby Standard Style (https://github.com/testdouble/standard)
standard: Run `standardrb --fix` to automatically fix some problems.
  /Users/code/cli.rb:31:23: Style/Semicolon: Do not use semicolons to terminate expressions.